nodejs/eks/cni/aws-k8s-cni.yaml

```diff
@@ -81,15 +81,20 @@ spec:
       tolerations:
         - operator: Exists
       containers:
-        - image: 602401143452.dkr.ecr.us-west-2.amazonaws.com/amazon-k8s-cni:v1.5.0
```

Comment:
Does this now march any released version of this yaml from the source?  Would love to keep this identical to some released version if possible.

Reply:
The upstream YAML manifest and ours match. Below is the diff between upstream and ours.
The main differences lie in:

- Removing the image, and `AWS_VPC_K8S_CNI_LOGLEVEL` as we set those above in `cni.ts`
- Enabling the use of the readiness & liveness probes since these are omitted upstream as this healthz endpoint is only available on CNI images >= `v1.5.2` which we use now.

```
$ diff -u upstream-aws-k8s-cni.yaml aws-k8s-cni.yaml
--- upstream-aws-k8s-cni.yaml   2019-08-08 11:28:15.838079451 -0700
+++ aws-k8s-cni.yaml    2019-08-07 19:12:21.888614038 -0700
@@ -81,23 +81,20 @@
       tolerations:
         - operator: Exists
       containers:
-        - image: 602401143452.dkr.ecr.us-west-2.amazonaws.com/amazon-k8s-cni:v1.5.2
-          imagePullPolicy: Always
+        - imagePullPolicy: Always
           ports:
             - containerPort: 61678
               name: metrics
           name: aws-node
-          #readinessProbe:
-          #  exec:
-          #    command: ["/app/grpc_health_probe", "-addr=:50051"]
-          #  initialDelaySeconds: 5
-          #livenessProbe:
-          #  exec:
-          #    command: ["/app/grpc_health_probe", "-addr=:50051"]
-          #  initialDelaySeconds: 5
+          readinessProbe:
+            exec:
+              command: ["/app/grpc_health_probe", "-addr=:50051"]
+            initialDelaySeconds: 5
+          livenessProbe:
+            exec:
+              command: ["/app/grpc_health_probe", "-addr=:50051"]
+            initialDelaySeconds: 5
           env:
-            - name: AWS_VPC_K8S_CNI_LOGLEVEL
-              value: DEBUG
             - name: MY_NODE_NAME
               valueFrom:
                 fieldRef:
```